### DISCUSSION TITLE
Host type stuck on MKS

### DISCUSSION THREAD

**Discussion by patofoto** [OrcaSlicer/OrcaSlicer#1960] [Q&A] [answered]
I am running a Ender3v2 with Klipper. The host type is stuck on MKS and I cannot change it. Running a nightly. Latest. Anyone experience this too?.

<img width="843" alt="Screenshot 2023-08-30 at 5 17 49 PM" src="https://github.com/SoftFever/OrcaSlicer/assets/37857628/a0a20687-2ec2-4594-9cf3-abd1966da5e6">

**Comment by darco415** [comment]
is this the 1.7 ver?....if so , revert back to latest

**Reply by patofoto** [reply]
Thanks for answer. Yes 1.7.0-dev version. Too bleeding edge?. Will revert. Thanks.

**Reply by darco415** [reply, marked as answer]
seems to be working now,,,,
